packages/model-viewer/src/test/features/environment-spec.ts

```diff
@@ -238,33 +175,16 @@ suite('ModelViewerElementBase with EnvironmentMixin', () => {
         });
       });
 
-      suite('on an unlit model', () => {
```

Comment:
The reason we had them in the first place was because we used to have a special code path for them regarding setting the environment lighting. That's now gone as it's properly handled in three, so I figured there was no longer a reason for a special test on our side.